src/style.css

```diff
@@ -0,0 +1,132 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url("https://fonts.googleapis.com");
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+:root {
+  --text-color: #060b35;
+  --text-second-color: #616070;
+  --main-color: #00acdc;
+  --background-origin: #fff;
+}
+
+.card {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  align-items: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--text-color);
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  align-self: start;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-second-color);
+  margin-bottom: 16px;
+}
+
+.card__evaluation {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-second-color);
+}
+
+.card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: 16px;
+}
+
+.card__price-title {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--text-second-color);
+}
+
+.card__amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--text-color);
+}
+
+.card__button {
+  display: inline-flex;
+  justify-content: center;
+  background-color: var(--main-color);
+  border-radius: 5px;
+  width: 166px;
+  height: 40px;
+  align-items: center;
+  color: var(--background-origin);
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 14px;
+  text-transform: uppercase;
+}
+
+.card__button:hover {
+  border: 1px solid var(--main-color);
```

Comment:
adding border on hover causes layout shift, you may need to change only border color

Reply:
yeah that's why it causes layout shift on hover. Try it yourself, when you hover the button it will "jump"

hint: you can set border-color transparent to button and update it on hover